src/redux/books/bookSlice.js

```diff
@@ -1,55 +1,66 @@
-import { v4 as uuidv4 } from 'uuid';
+/* eslint-disable camelcase */
+import axios from 'axios';
```

Comment:
- [ ] Please make sure that  [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) is imported in your  project to make API calls :pray: